versions/3.0.4.md

```diff
@@ -3076,7 +3073,7 @@ However, the exact nature of such conversions are implementation-defined.
 
 ##### Examples
 
-For these examples, assume all schemas are in the entry OpenAPI document; for handling of `discriminator` in referenced documents see [Resolving Implicit Connections](#resolving-implicit-connections).
+For these examples, assume all schemas are in a single-document OpenAPI Description; for handling of `discriminator` in referenced documents see [Resolving Implicit Connections](#resolving-implicit-connections).
```

Comment:
I wanted to make a point that these are resolved like this in the entry document even if the OAD is multi-document.  I don't feel strongly about this one, but I did have a purpose with that wording.